front-end/src/components/CreateEditPostModal.tsx

```diff
@@ -117,15 +133,32 @@ export default function CreateEditPostModal(props: Props){
           }).catch((error: any) => {
             setShowError(true)
           })
+
+        //put to post_id
+        const uuid = generateUUID();
+        console.log(uuid)
+        AxiosWrapper.put(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser.authorId + "/posts/" + uuid + "/", data, props.loggedInUser)
+        .then((res: any) => {
+          handleRes(res)
+        })
       }
+
       else if(props.editFields !== undefined){
           AxiosWrapper.post(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser.authorId + "/posts/" + props.editFields.id + "/", data, props.loggedInUser)
           .then((res: any) => {
             handleRes(res)
           }).catch((err: any) => {
             setShowError(true)
           })
+
+          //put to post_id
+          const uuid = generateUUID();
+          AxiosWrapper.put(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser.authorId + "/posts/" + uuid + "/", data, props.loggedInUser)
+          .then((res: any) => {
+            handleRes(res)
+          })
```

Comment:
maybe there's something in the missing file, but we should only put to post_id when a user clicks a button to generate a UUID. if the user does not click the button, the uuid field should be empty and this put request should not be called.